flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -423,4 +423,22 @@ under the License.
 			</plugin>
 		</plugins>
 	</build>
+
+	<profiles>
+		<profile>
+			<id>hive-1</id>
+			<properties>
+				<hive.version>1.2.1</hive.version>
+				<hivemetastore.hadoop.version>2.6.0</hivemetastore.hadoop.version>
+			</properties>
+			<dependencies>
+				<dependency>
```

Comment:
Because the 1.2.1 hive-exec jar doesn't include this dependency, and we excluded it from hive-metastore. BTW is there any reason why we excluded it from hive-metastore?

Reply:
it's excluded because it's not used previously. I think we can remove this as well as the exclusion of it in hive-metastore. I will have open a followup PR to make hive-metastore 'provided'

Reply:
it's excluded because it's not used previously. ok, I will have open a followup PR to make hive-metastore 'provided' and remove this dependency